src/validation.js

```diff
@@ -407,6 +407,36 @@ const validation = {
 
     return options;
   },
+  /**
+   * Ensures the windows.vbs file path is valid and exists.
+   * Resolves any environment variables to absolute paths.
+   * If no valid path is presented, use the default.
+   *
+   * @example
+   * options = validateWindowsVBScript(options);
+   *
+   * @param  {object} options  User's options
+   * @return {object}          Validated or mutated user options
+   */
+  validateWindowsVBScriptPath: function (options) {
+    if (!options.windows) {
+      return options;
+    }
+
+    if (options.windows.VBScriptPath) {
+      options.windows.VBScriptPath = helpers.resolveWindowsEnvironmentVariables(options.windows.VBScriptPath);
```

Comment:
This should come first, as it will verify if the value exists and is a string, if it doesn't, then `options.windows.VBScriptPath` will equal `undefined`. Therefore, we should not pass `undefined` into `fs.existsSync`
```suggestion
    options.windows.VBScriptPath = helpers.resolveWindowsEnvironmentVariables(options.windows.VBScriptPath);

    if (options.windows.VBScriptPath) {
```